website/docs/configuring_clients.md

```diff
@@ -8,8 +8,155 @@ title: Configuring Clients
  
 ## iOS
 
-TBD
+Prerequisites: to be able to follow the instructions below, you must have [cocoapods](https://cocoapods.org/)
+ installed and working and you must have a [variant](./configuring_variants.md#ios) already configured.
+For instructions on how to setup [cocoapods](https://cocoapods.org/) see [here](https://guides.cocoapods.org/using/getting-started.html)
 
+### Create a new project
+
+Open `XCode` and create a new project selecting `iOS` and `Single View App`:
+
+ >![CreateSingleViewApp](./assets/ios/CreateSingleViewApp.png)
+
+Click `Next`. In the next page, insert the name of you application (in this example _Demo App_) and select your team.
+
+ >![CreateDemoApp](./assets/ios/DemoApp.png)
+
+Click on `Finish` and save the app in a folder of your choice. Take note of that folder since we are going to open
+a terminal there.
+
+:::note
+Take note of the folder you will save the project in, since you will need to go there with the terminal and run some command to install the required pods!
+:::
+
+### Add required capabilities
+
+Click on the project to show the screen below:
+
+ >![CreateDemoApp](assets/ios/project-config.png)
+
+Select the _Sign & Capabilities_ tab and click on the `+` button in the upper left corner to add the following capabilities:
+* Background Modes
+* Push Notifications
+
+:::warning
+To be able to add the _Push Notifications_ capability be sure you select the right team and that the team has an active membership.
+:::
+
+After you have added both capabilities your window should look as below:
+
+ >![CreateDemoApp](./assets/ios/capabilities.png)
+
+Now, be sure you click in `Remote notifications` under `Background Modes` to enable it:
+
+>![CreateDemoApp](./assets/ios/checkBackgroundModes.png)
+
+### Registering your app
+
+Before you can receive any _push notification_ the first thing needed is to register the app to the _Unified Push Server_.
+To be able to perform such operation, you will need the following information:
+  * The URL of the _Unified Push Server_ instance
+  * The ID of an iOS variant you previously created in UPS (to see how, look [here](./configuring_variants#ios)) 
```

Comment:
Hi Massimo, 
The link "here" does results in a 404 page not found message.

Reply:
Just tried it and it works.
How are you looking at the docs? Are you running the webiste locally with `npm run start` ?